README.md

```diff
@@ -0,0 +1,404 @@
+# killbill
```

Comment:
Let's clean this up to a minimum:

* Keep instructions on how to install the library and a small Getting Started
* Remove generic `Swagger Codegen` stuff and add instructions on how to regenerate it based on our codegen library
* Let's remove `Documentation for API Endpoints`  and the `docs/*` files, as it's redundant with Slate